internal/controller/s3utils_test.go

```diff
@@ -68,10 +69,11 @@ func (fakeObjectStoreGetter) ObjectStore(
 
 	objectStorer, ok := fakeObjectStorers[s3ProfileName]
 	if !ok {
-		objectStorer = fakeObjectStorer{
+		objectStorer = &fakeObjectStorer{
 			name:       s3ProfileName,
 			bucketName: s3StoreProfile.S3Bucket,
 			objects:    make(map[string]interface{}),
+			mu:         sync.Mutex{},
```

Comment:
We don't need to initialize the mutex, the zero value of a mutex is good, basically what you added here.